test/e2e/e2e_test.go

```diff
@@ -156,11 +156,25 @@ func TestChannelTracing(t *testing.T) {
 		})
 }
 
-// TestSmokePullSubscription makes sure we can run tests on PullSubscriptions.
-func TestSmokePullSubscription(t *testing.T) {
+// TestSmokePullSubscriptionV1 test we can create a v1 PullSubscription to ready state
+func TestSmokePullSubscriptionV1(t *testing.T) {
 	cancel := logstream.Start(t)
 	defer cancel()
-	SmokePullSubscriptionTestImpl(t, authConfig)
+	SmokePullSubscriptionTestHelper(t, authConfig, "v1")
+}
+
+// TestSmokePullSubscriptionV1beta1 test we can create a v1beta1 PullSubscription to ready state
+func TestSmokePullSubscriptionV1beta1(t *testing.T) {
+	cancel := logstream.Start(t)
+	defer cancel()
+	SmokePullSubscriptionTestHelper(t, authConfig, "v1beta1")
+}
+
+// TestSmokePullSubscriptionV1alpha1 test we can create a v1alpha1 PullSubscription to ready state
+func TestSmokePullSubscriptionV1alpha1(t *testing.T) {
```

Comment:
I will add some comment to indicate for each supported version of resources, we only write smoke tests

Reply:
if you can add that, it'd be great